### ISSUE TITLE
Broken container?

### ISSUE THREAD

**Issue by schvabodka-man** [frozenfoxx/rpi-calibre#1]
Hello, i'm building this, running and simply getting
`docker: Error response from daemon: oci runtime error: container_linux.go:247: starting container process caused "exec: \"calibre-server\": executable file not found in $PATH".
`

**Comment by frozenfoxx** [comment]
Yes, at the moment this isn't functional. When last I looked into this I was having some severe issues with getting Python to build it appropriately and do not currently have the time to look into it. I will see if there's anything I can do recently but at the moment I'm accepting pull requests.
